cypress/integration/AnnualPlan/ContentBlockAPLink/contentBlock.js

```diff
@@ -31,6 +31,6 @@ Then('the PDF download link is not displayed', () => {
 });
 
 When('user clicks on top {string} button', (hrefAttr) => {
-    cy.get("div[class='ap-file-block pdf']").find('a').trigger('click', {followRedirect:false});
+    cy.get("div[class='ap-file-block pdf']").find('a').trigger('click', { followRedirect: false });
```

Comment:
just revert formatted change here - it's confusing to see a difference

Reply:
Done